app/src/test/java/com/futurice/freesound/feature/search/DefaultSearchDataModelTest.java

```diff
@@ -73,78 +72,76 @@ public void querySearch_emitsSingleUnit_whenQuerySearchSuccessful() {
         new Arrangement()
                 .withSearchResultsFor(QUERY, dummyResults());
 
-        TestSubscriber<Unit> ts = testSubscribe(defaultSearchDataModel.querySearch(QUERY));
+        TestSubscriber<Unit> ts = defaultSearchDataModel.querySearch(QUERY).test();
 
-        assertThat(ts).hasNoErrors()
-                      .hasReceivedValue(Unit.DEFAULT)
-                      .hasCompleted();
+        ts.assertValue(Unit.DEFAULT);
     }
 
     @Test
     public void querySearch_emitsError_whenQuerySearchErrors() {
         final Exception expected = new Exception();
         new Arrangement().withSearchResultError(expected);
 
-        TestSubscriber<Unit> ts = testSubscribe(defaultSearchDataModel.querySearch("query"));
+        TestSubscriber<Unit> ts = defaultSearchDataModel.querySearch("query").test();
 
-        assertThat(ts).hasError(expected);
+        ts.assertError(expected);
     }
 
     @Test
     public void querySearch_returnsResults_whenQuerySearch_withEmptyTerm() {
         final SoundSearchResult result = dummyResults();
         new Arrangement()
                 .withSearchResultsFor("", result);
-        TestSubscriber<Option<List<Sound>>> ts = testSubscribe(
-                defaultSearchDataModel.getSearchResults());
+        TestObserver<Option<List<Sound>>> ts =
```

Comment:
In new version RxJava 1.x, `TestObserver<T>` has been deprecated. I'll have to look into whether it has a future in RxJava2. Have you heard/know anything about that?